web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -8,8 +8,7 @@
       <v-row class="mx-2 my-2 mb-0">
         <v-col>
           <h1 :class="`font-weight-light ${$vuetify.breakpoint.xs ? 'text-h6' : ''}`">
-            {{ meta.name }}
-            <ShareDialog />
+            <ShareDialog />{{ meta.name }}
```

Comment:
```suggestion
            <ShareDialog />
            {{ meta.name }}
```
Minor nitpick, is there any issue with doing this? It's easier to read that way imo (unless of course, that messes up the formatting)